Add sort option and stream key listing to inspector

ksauzz opened the pull request.

shino commented on priv/tools/internal/riak_cs_inspector.erl:
```diff
@@ -84,10 +84,7 @@
-%-define(sort_opt_spec, [{sort, $s, "sort", undefined,
-                            %"determine to sort results (not implemented yet)"}]).
--define(sort_opt_spec, []).
+-define(sort_opt_spec,   [{sort, $s, "sort", undefined, "Determine to sort results instead of streaming list keys"}]).
```
At 4th item, Instead of `undefined`, `{boolean, false}` may be better.
The help message can be more simple? For example "Sort the results".

shino commented on priv/tools/internal/riak_cs_inspector.erl:
```diff
@@ -259,41 +261,110 @@ cs_bucket_info(RiakBucket, KnownNames) ->
+            fun pb_stream_list_keys/3
+    end.
+
+% For Backward compatibility. Is this needed?
```
I think this function can be removed. :hear_no_evil:

shino commented on priv/tools/internal/riak_cs_inspector.erl:
```diff
@@ -259,41 +261,110 @@ cs_bucket_info(RiakBucket, KnownNames) ->
+               end,
+    ListFun(RiakcPid, "moss.buckets", PrintFun).
+
+lookup_listbucket_fun(Opts) ->
```
It's better to place this function and `lookup_listkey_fun/1` after `%% Riak Utilities` comment line.

shino commented on priv/tools/internal/riak_cs_inspector.erl:
```diff
@@ -259,41 +261,110 @@ cs_bucket_info(RiakBucket, KnownNames) ->
+               end,
+    ListFun(RiakcPid, ManifestBucketName, PrintFun).
+
+pb_list_buckets(RiakcPid, PrintFun) ->
```
Also, from this function to `print_stream_list()` can be moved to `%% Riak Utilities` part.

shino commented:
Great functionality! Works as advertised.

I added some comments but only superficial ones. Will :+1: after fix.

ksauzz commented:
Thanks. I updated the items you mentioned. Also I moved pb_xxxxx functions to Riak Utilities section, and removed list_bucket/1 I had left for backward compatibility such as list_objects/2.

shino commented:
@borshop merge
